gpiocontrol/source/android/app/src/main/java/org/kaaproject/kaa/examples/gpiocontol/DevicesListActivity.java

```diff
@@ -57,18 +56,13 @@
 
     private final String TAG = DevicesListActivity.class.getSimpleName();
```

Comment:
Please, provide some comments to this class. What does he do? Simlar to other classes - please, add comments